### ISSUE TITLE
404 on tagsinput.css

### ISSUE THREAD

**Issue by Salamandar** [spiral-project/ihatemoney#752]
Hi,

I don't know if it's problematic (there's no visual issue), but on my nginx logs i found that : 
```
2021/05/22 18:29:41 [error] 20226#20226: *51687 open() "/var/www/ihatemoney/venv/lib/python3.7/site-packages/ihatemoney/static/css/tagsinput.css" failed (2: No such file or directory), client: 2001:861:3381:6960:2825:a6be:f517:1d5a, server: salamandar.fr, request: "GET /ihatemoney/static/css/tagsinput.css HTTP/2.0", host: "salamandar.fr", referrer: "https://salamandar.fr/ihatemoney/static/css/main.css"
```

I searched around, and in fact tagsinput.css is not installed with pip.

**Comment by zorun** [comment]
This `include tagsinput.css` was wrongly backported to the 4.1 stable branch, it's not actually used there. It's a new feature of the invite page that will only be present in newer versions.

So in short: it's completely harmless.

I've fixed in the `stable-4.1` branch for good measure, but I don't think we will be doing a new 4.1.X release before the next major version 5.

**Comment by Salamandar** [comment]
OK !! Thanks a lot for the fix :D
